src/main/java/hudson/plugins/cppncss/parser/Statistic.java

```diff
@@ -16,10 +18,16 @@
  * @author Shaohua Wen
  * @since 25-Feb-2008 21:33:40
  */
+@SuppressFBWarnings(value = "SE_NO_SERIALVERSIONID", justification = "Not used in XStream")

```

Comment:
You mean, not used in Remoting? XStream ignores `serialVersionUID`s.

Maybe easier to just add the constant anyway?

Reply:
Will replace to "Ignored in XStream"